FoodplannerDataAccessSql/Migrations/11_AddOTPTable.cs

```diff
@@ -0,0 +1,31 @@
+﻿using FluentMigrator;
+namespace FoodplannerDataAccessSql.Migrations
+{
+    [Migration(11)]
```

Comment:
This migration should be 12 instead of changing existing migrations:
```suggestion
    [Migration(12)]
```

Reply:
nu uh, det er den anden jeg har ædnret til 12